resources/views/layout.blade.php

```diff
@@ -25,6 +25,7 @@
     {!! HTML::style('//fonts.googleapis.com/css?family=Yellowtail') !!}
     {!! HTML::style('//fonts.googleapis.com/css?family=Josefin+Slab:600') !!}
     {!! HTML::style(elixir("css/app-all.css")) !!}
+    @yield('styles')
```

Comment:
I'll leave that yield in there; might be useful later on for testing, etc. I have moved the styling that was in there into app-all.